### DISCUSSION TITLE
Custom image settings start and wait 1s

### DISCUSSION THREAD

**Discussion by eqfae** [zbm-dev/zfsbootmenu#653] [Questions]
Build with github action
What needs to be changed in the source code?

**Comment by classabbyamp** [comment]
what?

**Reply by eqfae** [reply]
> what?

Change 10s to 1s

**Comment by zdykstra** [comment]
That's controlled by the `zbm.timeout` kernel parameter.  https://docs.zfsbootmenu.org/en/v2.3.x/man/zfsbootmenu.7.html#command-line-parameters

**Reply by eqfae** [reply]
efibootmgr test work
refind tried all kinds of things but it didn't work 
Dual systems require refind

**Reply by eqfae** [reply]
zfs set org.zfsbootmenu:commandline="ro quiet loglevel=0 zbm.timeout=1" zroot/linux
Is this so?

**Reply by zdykstra** [reply]
`org.zfsbootmenu:commandline` is for the OS that you're booting. Use `zbm-kcl` to edit the command line embedded in your ZFSBootMenu EFI file.
